### DISCUSSION TITLE
@requires is triggered by a single "Middleware", depending on its order, when reading the script

### DISCUSSION THREAD

**Discussion by wnesbv** [encode/starlette#2277] [Potential Issue]
https://github.com/wnesbv/starlette-example/blob/main/main.py
https://github.com/wnesbv/starlette-example/blob/main/account/middleware.py
https://github.com/wnesbv/starlette-example/blob/main/auth_privileged/middleware.py
https://github.com/wnesbv/starlette-example/blob/main/templates/auth/details.html

**Comment by Kludex** [comment]
If you want to be helped, I recommend creating a minimal, reproducible example, and explaining it in an easy way.

Also, please stop spamming discussions.

**Reply by wnesbv** [reply]
Well, you don't have time to figure out your mistakes - just say so... What kind of boss are you - to point out - one laugh... Dear to you by addresses..

**Comment by Kludex** [comment]
I've locked the discussion, since the author's behavior doesn't seem appropriate.
